third_party/CMakeLists.txt

```diff
@@ -63,11 +63,9 @@ add_subdirectory(glfw)
 add_subdirectory(glm)
 
 #### VulkanMemoryAllocator
-include_directories(VulkanMemoryAllocator/src)
-include_directories(${Vulkan_INCLUDE_DIRS})
 add_library(VulkanMemoryAllocator STATIC "VulkanMemoryAllocatorBuild.cpp")
-target_include_directories(VulkanMemoryAllocator INTERFACE VulkanMemoryAllocator/src)
-target_include_directories(VulkanMemoryAllocator PRIVATE "${SCIN_EXT_INSTALL_DIR}/include")
+target_include_directories(VulkanMemoryAllocator PUBLIC VulkanMemoryAllocator/src)
+target_link_libraries(VulkanMemoryAllocator PUBLIC vulkan)
```

Comment:
i think this dependency should be public, actually. since VulkanMemoryAllocator is static, it won't be linked with libvulkan when it's built; so anything linking against VulkanMemoryAllocator should link against vulkan too. this also makes the include directories a little more generic